sdks/python/apache_beam/transforms/core.py

```diff
@@ -1855,7 +1865,7 @@ def __init__(self, value):
       value: An object of values for the PCollection
     """
     super(Create, self).__init__()
-    if isinstance(value, string_types):
+    if isinstance(value, (unicode, str)):
```

Comment:
Should we add `bytes` here?

Reply:
Based on the requirements of value (possible to, I think we should actually check if value is an iterable:
`hasattr(values,'__iter__')` which also fails for string_types

Reply:
The intention of this line is to prohibit string types from being returned where we expect an iterable of items.  Strings technically are iterable (and return individual characters), so we want to prevent them from being returned accidentally (e.g., a user may intend to return a single string, but we don't want to interpret it as its individual characters).  In both Python 2 and Python 3, string types are iterable, so I think we should add `bytes` to this list of "blacklisted" return types.